certified-connectors/Entersoft/apiProperties.json

```diff
@@ -0,0 +1,39 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "a1",
```

Comment:
@entersoftsa Please update the client id value to "[[DummyValue]]" as it is not allowed to expose sensitive information in the open source repo. You can keep the actual value during submission in the certification portal.

Reply:
Hi to all,

Yes, I am aware about these limitations. Actually, I came across then when I started the dev of the connector as I could not create a custom connector due to the Num of operations limitation. Thus I revised the API’s OpenApiDefinition through the Swagger configuration, so that I am exposing less than 128 operations by selecting the most appropriate ones. The same applies for the Definitions.

So, the answer is NO, Entersoft Connectors does NOT exceed these limitations.

Thanks